crates/eips/src/eip7702/auth_list.rs

```diff
@@ -142,6 +145,7 @@ impl Deref for RecoveredAuthorization {
 ///
 /// The wrapper type is used for RLP encoding and decoding.
 #[derive(Default, Debug, Copy, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct OptionalNonce(Option<u64>);
```

Comment:
do we want to serialize this as list?

Reply:
I am not sure, this would depend on jsonrpc and how it is defined there. Are you okay with leaving it as it is, until more info is known?